test/Sema/accessibility.swift

```diff
@@ -1471,7 +1471,11 @@ public class DerivedFromInternalConcreteGenericComposition : InternalConcreteGen
   public func publicReq() {}
 }
 
-// FIXME: rdar://104987455 should have expected note and error 'class cannot be declared public because its superclass is internal'
```

Comment:
turns out this was expected; added a test using `fileprivate` typealias from `internal` decls to illustrate this